firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1972,6 +1974,22 @@ class BrowserViewController: UIViewController,
         presentSheetWith(viewModel: viewModel, on: self, from: button)
     }
 
+    func presentNewTabLongPressActionSheet(from view: UIView) {
+        let oneTapNewTabEnabled = featureFlags.isFeatureEnabled(.toolbarOneTapNewTab, checking: .buildOnly)
+        guard presentedViewController == nil, oneTapNewTabEnabled else { return }
```

Comment:
should this never return early from a user perspective since this shouldn't be called at all? If so, can what do you think about adding a warning or info log here?

Reply:
hmm, would you mind clarifying 😅

Reply:
Could totally add a log here, but come to think about it, I wonder if this guard is even necessary. Maybe just a safety check? I feel like if you are able to trigger the long press while another view controller is presented would indicate we had other problems

Reply:
Seems like it was removed so resolving this, if its a safety check we can always add a log with a warning in case something fishy is going on. However, if we don't need to check at all, then def can remove.